src/utils.js

```diff
@@ -28,7 +28,11 @@ export const btcToSats = value => Math.round(parseFloat(value) * 100000000)
 
 export const satsToBtc = value => parseInt(value, 10) / 100000000
 
-export const valueToUnit = (value, unit) => {
+export const valueToUnit = (value, unit, showValue = true) => {
```

Comment:
I'd say we should then default this one to false as well:

```suggestion
export const valueToUnit = (value, unit, showValue = false) => {
```

Reply:
Yeah that's a good point! 👍  I initially thought I don't want to change it for other pages but this function is not used anywhere else than on the main wallet page so we can definitely change the default here. 

Updated. ✅